src/styles/fonts/_fontawesome.scss

```diff
@@ -0,0 +1,2199 @@
+/*!
```

Comment:
@mahboob-awan agreed. I think slack or skype would be ideal play to discuss these things.